src/Symfony/Bundle/WebServerBundle/Command/ServerCommand.php

```diff
@@ -20,11 +20,4 @@
  */
 abstract class ServerCommand extends Command
```

Comment:
Shouldn't it be removed (and marked deprecated in 3.x) instead? It doesn't really serve a purpose anymore, does it?